edpcmentoring/edpcmentoring/wsgi.py.mws3

```diff
@@ -0,0 +1,30 @@
+"""
+WSGI config for edpcmentoring project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/howto/deployment/wsgi/
+
+The whitenoise configuration is taken from
+
+    https://devcenter.heroku.com/articles/django-assets
+
+It is what allows the application to serve its own static files in production.
+
+"""
+
+#start for mws3 hosts:
+import sys
+sys.path.append('/var/www/default/admindir/edpcmentoring/edpcmentoring')
+sys.path.append('/var/www/default/admindir/edpcmentoring/env/lib/python2.7/site-packages')
```

Comment:
Colour me crazy but doesn't virtualenv set this one up for us via the ``activate`` script?